controllers/user.js

```diff
@@ -101,3 +100,69 @@ export const signup = async (req, res) => {
     console.log(error);
   }
 };
+
+/*
+<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+*/
+
+// For User Profile
+import mongoose from 'mongoose';
+
+// Fetch all posts
+export const getUsers = async (req, res) => {
+  try {
+    const allUsers = await User.find();
+
+    console.log(allUsers);
+
+    res.status(200).json(allUsers);
+  } catch (error) {
+    res.status(404).json({ message: error });
+  }
+};
+
+// Fetch a single post
+export const getSingleUser = async (req, res) => {
+  try {
+    const user = await User.findById(req.params.id);
+    res.status(200).json(user);
+  } catch (error) {
+    res.status(404).json({ message: error });
+  }
+};
+
+// Update a Post
```

Comment:
just need to adjust the comment change `Post` to `User` ? To avoid any confusion ?